docs/content/latest/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -12,67 +12,170 @@ isTocNested: true
+1. The root certificate (`ca.cert`).
+2. The node certificate (`node.ip_address.crt`). 
+3. The node private key (`node.ip_address.key`). 
 
-## Enable TLS using platform generated certificates
+The platform retains the root certificate and the root private key for all interactions with the cluster.
 
-1. Create a new universe **Universes -> Create Universe**.
-2. Configure the universe as desired.
-3. Based on your needs, select **Enable Node-to-Node TLS** and **Enable Client-to-Node TLS**.
-4. Choose an existing certificate from the **Root Certificate** drop-down list or leave as default “Create new certificate” if a new one is desired to be used.
-5. Create the universe.
+### How to Use Platform-Generated Certificates to Enable TLS
 
-To view the certificate, select the drop-down list in the top-right corner of the Yugabyte Platform console and click **Certificates**.
+You can enable TLS using certificates generated by Yugabyte Platform, as follows:
 
-## Enable TLS using a self-signed, user-provided certificate
+- Create a new universe via **Universes > Create Universe** and then configure it.
+- Based on your requirements, select **Enable Node-to-Node TLS** and **Enable Client-to-Node TLS**.
+- Choose an existing certificate from the **Root Certificate** list or create a new certificate by accepting the default option **Create new certificate**.
 
-1. In the top-right corner of the Yugabyte Platform console, click the drop-down list (with the profile icon) and select **Certificates**.
-2. Click **Add Certificate**. The **Add Certificate** dialog appears.
-3. Click **Upload Certificate File**, browse to the root certificate file (`<file-name>.crt`), and upload it.
-4. Click **Upload Key File**, browse to the root certificate file (`<file-name>.key`), and upload it.
-5. In **Name Certificate**, enter a meaningful name.
-6. In **Expiration Date**, specify the expiration date of the certificate. The expiration date can be found by running the following command: `openssl x509 -in <root crt file path> -text -noout` and then looking for the **Validity Not After** date.
-7. Click **Add**. The certificate is available.
-8. Go to **Universes -> Create Universe**. The **Create Universe** dialog appears.
-9. Configure the universe as desired.
-10. Based on your needs, select **Enable Node-to-Node TLS** and **Enable Client-to-Node TLS**.
-11. Choose an existing certificate from the **Root Certificate** drop-down list and then select the certificate that was uploaded.
-12. Create the universe.
+To view the certificate, use the list located at the top-right corner of the Yugabyte Platform console and click **Certificates**.
```

Comment:
@lizayugabyte Can you change this line to this? 
To view the certificate, 
1. Navigate to Configs, select the Security tab, and then select the Encryption in Transit tab.

2. Select the Self Signed tab.